packages/discord.js/src/index.js

```diff
@@ -151,7 +150,6 @@ exports.InteractionCollector = require('./structures/InteractionCollector');
 exports.InteractionResponse = require('./structures/InteractionResponse');
 exports.InteractionWebhook = require('./structures/InteractionWebhook');
 exports.Invite = require('./structures/Invite');
-exports.InviteStageInstance = require('./structures/InviteStageInstance');
 exports.InviteGuild = require('./structures/InviteGuild');
```

Comment:
This should be removed too:
https://github.com/discordjs/discord.js/blob/9aed9414c080ea7cbe0cec407d9a71bef903c7d5/packages/discord.js/typings/rawDataTypes.d.ts#L151